src/Profiles/Lev/ANTPLUS_LevPrivateDefines.h

```diff
@@ -0,0 +1,24 @@
+#ifndef ANTPLUS_LEVPROFILEPRIVATEDEFINES_h
+#define ANTPLUS_LEVPROFILEPRIVATEDEFINES_h
+
+/* Channel Config */
+#define ANTPLUS_LEV_CHANNELTYPE CHANNEL_TYPE_BIDIRECTIONAL_RECEIVE
```

Comment:
Lets make this ANTPLUS_LEV_DISPLAY_CHANNELTYPE so we know which side of the profile we are talking about

Reply:
Changed